int64 inp
---
bool fb
